### PR TITLE
Added usage examples to some APIs

### PR DESCRIPTION
**Pull request by msteknoadam**
Added usage examples to these APIs
- image.random_flip_up_down
- image.flip_up_down
- image.random_flip_left_right
- image.flip_left_right
- image.transpose
- image.random_brightness
- image.random_contrast
- image.random_hue
- image.random_jpeg_quality
- image.random_saturation

### CODE REVIEW

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -337,6 +337,13 @@ def random_flip_up_down(image, seed=None):
     A tensor of the same type and shape as `image`.
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
   
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -359,6 +366,13 @@ def random_flip_left_right(image, seed=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -425,6 +439,13 @@ def flip_left_right(image):
...
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 4]
```diff
@@ -446,6 +467,13 @@ def flip_up_down(image):
...
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 5]
```diff
@@ -601,6 +629,13 @@ def transpose(image, name=None):
...
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 6]
```diff
@@ -1537,6 +1572,13 @@ def random_brightness(image, max_delta, seed=None):
 
   Raises:
     ValueError: if `max_delta` is negative.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 7]
```diff
@@ -1564,6 +1606,13 @@ def random_contrast(image, lower, upper, seed=None):
 
   Raises:
     ValueError: if `upper <= lower` or if `lower < 0`.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 8]
```diff
@@ -1899,6 +1948,13 @@ def random_hue(image, max_delta, seed=None):
 
   Raises:
     ValueError: if `max_delta` is invalid.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 9]
```diff
@@ -1977,6 +2033,13 @@ def random_jpeg_quality(image, min_jpeg_quality, max_jpeg_quality, seed=None):
 
   Raises:
     ValueError: if `min_jpeg_quality` or `max_jpeg_quality` is invalid.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 10]
```diff
@@ -2068,6 +2131,13 @@ def random_saturation(image, lower, upper, seed=None):
 
   Raises:
     ValueError: if `upper <= lower` or if `lower < 0`.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    
```

**Review by kyscg — CHANGES_REQUESTED** [review]
It is not necessary to import tensorflow everywhere

**Comment by mihaimaruseac** [comment]
Please reopen against master, not against release branches

**Pull request closed without merging**